test/integ/test_connection.py

```diff
@@ -52,7 +52,6 @@
 except ImportError:  # Keep olddrivertest from breaking
     ER_FAILED_PROCESSING_QMARK = 252012
 
-
```

Comment:
was this removed by linter? if not please add this back

Reply:
reverted